client/google-search-app/src/app/search-form/search-form.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="parent">
+
+  <div class="center">
+    <img src="assets/images/googlelogo_color_272x92dp.png" alt="Google Search branding logo">
+
+    <input required="required">
+    <button name="google-search">Google Search</button>
```

Comment:
these text can be localized in future PRs. #62

Reply:
ok